<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_go_to_main">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>홈으로</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_title">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumGothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>관리자 페이지</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>681</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lbl_1">
      <property name="font">
       <font>
        <family>NanumGothic</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>총 매출액 :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_total">
      <property name="font">
       <font>
        <family>NanumGothic</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="graph_widget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>269</y>
     <width>681</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
